eap73-rhel8-byos-multivm/scripts/jbosseap-setup-redhat.sh

```diff
@@ -126,42 +133,30 @@ echo "Copy the standalone-azure-ha.xml from EAP_HOME/doc/wildfly/examples/config
-$EAP_HOME/wildfly/bin/standalone.sh -bprivate $IP_ADDR -b $IP_ADDR -bmanagement $IP_ADDR --server-config=standalone-azure-ha.xml -Djboss.jgroups.azure_ping.storage_account_name=$STORAGE_ACCOUNT_NAME -Djboss.jgroups.azure_ping.storage_access_key=$STORAGE_ACCESS_KEY -Djboss.jgroups.azure_ping.container=$CONTAINER_NAME -Djava.net.preferIPv4Stack=true | adddate >> /var/log/jbosseap.install.log 2>&1 &
-sleep 20
-####################### 
-
-####################### Setup JBoss server to start on boot
-echo "export IP_ADDR=\$(hostname -I)" >> /bin/jbossservice.sh
-echo "export EAP_HOME="/opt/rh/eap7/root/usr/share"" >> /bin/jbossservice.sh
-
-echo "export STORAGE_ACCOUNT_NAME=$STORAGE_ACCOUNT_NAME" >> /bin/jbossservice.sh
-echo "export STORAGE_ACCESS_KEY=$STORAGE_ACCESS_KEY" >> /bin/jbossservice.sh
-echo "export CONTAINER_NAME=$CONTAINER_NAME" >> /bin/jbossservice.sh
-
-echo "\$EAP_HOME/wildfly/bin/standalone.sh -bprivate \$IP_ADDR -b \$IP_ADDR -bmanagement \$IP_ADDR --server-config=standalone-azure-ha.xml -Djboss.jgroups.azure_ping.storage_account_name=\$STORAGE_ACCOUNT_NAME -Djboss.jgroups.azure_ping.storage_access_key=\$STORAGE_ACCESS_KEY -Djboss.jgroups.azure_ping.container=\$CONTAINER_NAME -Djava.net.preferIPv4Stack=true &" >> /bin/jbossservice.sh
-chmod +x /bin/jbossservice.sh
-
-yum install cronie cronie-anacron | adddate >> /var/log/jbosseap.install.log 2>&1
-service crond start | adddate >> /var/log/jbosseap.install.log 2>&1
-chkconfig crond on | adddate >> /var/log/jbosseap.install.log 2>&1
-echo "@reboot sleep 90 && /bin/jbossservice.sh" >>  /var/spool/cron/root
-chmod 600 /var/spool/cron/root
+echo "Updating standalone-azure-ha.xml" | adddate >> /var/log/jbosseap.install.log
+echo -e "\t stack UDP to TCP"           | adddate >> /var/log/jbosseap.install.log
+echo -e "\t management:inet-address"    | adddate >> /var/log/jbosseap.install.log
+echo -e "\t public:inet-address"        | adddate >> /var/log/jbosseap.install.log
+echo -e "\t private:inet-address"       | adddate >> /var/log/jbosseap.install.log
+echo -e "\t webservices:wsdl-host"      | adddate >> /var/log/jbosseap.install.log
+
+$EAP_HOME/wildfly/bin/jboss-cli.sh --echo-command \
+'embed-server --std-out=echo  --server-config=standalone-azure-ha.xml',\
+'/subsystem=jgroups/channel=ee:write-attribute(name="stack", value="tcp")',\
+'/interface=management:write-attribute(name=inet-address, value="${jboss.bind.address.management:0.0.0.0}")',\
+'/interface=public:write-attribute(name=inet-address, value="${jboss.bind.address:0.0.0.0}")',\
+'/interface=private:write-attribute(name=inet-address, value="${jboss.bind.address.private:0.0.0.0}")',\
+'/subsystem=webservices:write-attribute(name=wsdl-host, value="${jboss.bind.address:0.0.0.0}")' | adddate >> /var/log/jbosseap.install.log 2>&1
```

Comment:
On the other hand, this one should be set to Azure VM public IP if this should work outside Azure cloud. I think we should not set this one and leave default there.

Reply:
done